<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
    <line>
      <![CDATA[C:\Users\rockstr\eclipse-workspace\exeeell\src\test\resource\Failed_XML\2.xml]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="1841" started-at="2020-03-21T05:12:31Z" finished-at="2020-03-21T05:12:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1841" started-at="2020-03-21T05:12:31Z" finished-at="2020-03-21T05:12:33Z">
      <class name="TestRun.RetryFailedTest">
        <test-method status="PASS" signature="TestRun()[pri:0, instance:TestRun.RetryFailedTest@1832929]" name="TestRun" duration-ms="1812" started-at="2020-03-21T05:12:31Z" finished-at="2020-03-21T05:12:33Z">
          <reporter-output>
            <line>
              <![CDATA[C:\Users\rockstr\eclipse-workspace\exeeell\src\test\resource\Failed_XML\2.xml]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestRun -->
      </class> <!-- TestRun.RetryFailedTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
